### DISCUSSION TITLE
Gimbal FPV camera

### DISCUSSION THREAD

**Discussion by Taboaco** [iNavFlight/inav#7745] [Q&A]
Hy guys, is it possible to enable a kind of mechanical stabilization to the FPV camera with a gimbal style but just so that it cuts quick fast movements limited to something like 10°?
Just so it reduces the turbulence shaking but not reduces the feel of the plane, so we can still fly FPV but have a better “comfort” ?

**Comment by kaliblazin707** [comment]
Like setting up servos on "stabilized gimble" in mixer?

**Reply by Taboaco** [reply]
Something like that, but acting like a dumper, like a shock absorber reducing the effects of the shakes due turbulence
